Documentation/ApiOverview/FormEngine/Rendering/_ImportDataControl.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\FormEngine\FieldControl;
+
+use TYPO3\CMS\Backend\Form\AbstractNode;
+
+class ImportDataControl extends AbstractNode
+{
+    private string $langFile = 'LLL:EXT:my_extension/Ressources/private/Languages/locallang_db.xlf';
```

Comment:
```suggestion
    private string $langFile = 'LLL:EXT:my_extension/Ressources/private/Languages/locallang_db.xlf';

```

For better readability: add an empty line.